### DISCUSSION TITLE
Changing username

### DISCUSSION THREAD

**Discussion by dushmanta05** [EddieHubCommunity/LinkFree#6696] [Q&A]
Hi there! I've already created my LinkFree account by adding "username".json file, now I want to change my username, so can you please confirm if I can change the filename and create a PR, and that'll do the magic? Thanks.

**Comment by dushmanta05** [comment]
I've found the solution, so I'm updating the answer myself.

The username.json files can only be named after GitHub username, so in order to change the BioDrop (formerly LinkFree) username, we need to change the GitHub username.

**Reply by eddiejaoude** [reply]
Sorry I missed the notification. Yes your BioDrop username must match your GitHub username for the moment

**Reply by dushmanta05** [reply]
Hi, no worries, You yourself helped me by replying to this query on Twitter. I just forgot to update it here, so I answered it.
